src/components/DelegateMultipleButton.jsx

```diff
@@ -187,6 +192,14 @@ const DelegateMultipleButton = props => {
     setLoadingDonations(false);
   };
 
+  const isLimitedDelegateCount = () => {
+    if (props.milestone && props.milestone.isCapped) {
+      return totalDonations > delegations.length && ifDelegationLimited;
```

Comment:
Imagine this case:
I have a campaign with 20 donations of 1 DAI value from different donators (so each one will be in different pledge), and a milestone which needs 16 DAI. Then `ifDelegationLimited` will be false and the message is not shown while it's incorrect

Reply:
In this case ifDelegationLimited is true since newAmount considers config.donationDelegateCountLimit